Testing/Readme.md

```diff
@@ -0,0 +1,424 @@
+
+**Configuration options**: Looking into the tenant there are multiple options in allowing and restricting different groups of users to automatically admitted into a meeting. There are two config options that would fail the SCB and others that would pass the SCB.
+
+![Teams 1.3 Config](/images/Teams-1-3.png)
+
+This is an automated functional test case for non-compliant and compliant config options:
+
+```
+  - PolicyId: MS.TEAMS.1.3v1
+    TestDriver: RunScuba
+    Tests:
+      - TestDescription: MS.TEAMS.1.3v1 Non-compliant - Dialin bypass lobby; Everyone is autoadmitted user
+        Preconditions:
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              Identity: Global
+              AllowPSTNUsersToBypassLobby: true
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              Identity: Global
+              AutoAdmittedUsers: EveryOne
+        Postconditions: []
+        ExpectedResult: false
+      - TestDescription: MS.TEAMS.1.3v1 Non-compliant - Dialin bypass lobby; Everyone not auto admitted user
+        Preconditions:
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              Identity: Global
+              AllowPSTNUsersToBypassLobby: true
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              AutoAdmittedUsers: EveryoneInCompany
+              Identity: Global
+        Postconditions: []
+        ExpectedResult: false
+      - TestDescription: MS.TEAMS.1.3v1 Compliant - Dialin not bypass lobby Everyone not auto admitted user
+        Preconditions:
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              Identity: Global
+              AllowPSTNUsersToBypassLobby: false
+          - Command: Set-CsTeamsMeetingPolicy
+            Splat:
+              AutoAdmittedUsers: EveryoneInCompany
+              Identity: Global
+        Postconditions: []
+        ExpectedResult: true
+```
+
+## Nightly Functional Tests ##
+
+A GitHub workflow is used to execute multiple product functional tests every night, and they are being run for multiple product-tenant combinations. Development teams should work with operational team to debug any issues found during nightly and/or weekly functional tests. The current list of tests can be found in a Slack canvas. 
+
+## Troubleshooting ##
+
+### Issues with installing Pester
+If the Pester module is pre-installed and pre-loaded on your system, you may see following message when you attempt to install it:
+
+```
+WARNING: The version '5.5.0' of module 'Pester' is currently in use. Retry the operation after closing the
+applications.
+```
+
+To Resolve above issue, try the following:
+- Close the PowerShell session
+- Open Task Manager to find any background instances of PowerShell and close them all 
+- Uninstall all versions of Pester using following command
+```
+Uninstall-Module -Name Pester -Force -AllVersions
```

Comment:
Recommend indenting, like line 369 below, for consistency.

Reply:
done